cas-server-core-logging/src/main/java/org/jasig/cas/util/CasLoggerContextInitializer.java

```diff
@@ -35,109 +25,44 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(CasLoggerContextInitializer.class);
 
-    private ServletContext context;
-
-    private ServletContextListener loggerContext;
-
-    @Value("${log4j.config.package:org.apache.logging.log4j.web}")
```

Comment:
They weren't in cas.properties, I can check if they are elsewhere in documentation